api/desecapi/models.py

```diff
@@ -464,78 +298,39 @@ class RRset(models.Model, mixins.SetterMixin):
     class Meta:
         unique_together = (("domain", "subname", "type"),)
 
-    def __init__(self, *args, **kwargs):
-        self._dirties = set()
-        super().__init__(*args, **kwargs)
-
-    def setter_domain(self, val):
-        if val != self.domain:
-            self._dirties.add('domain')
-
-        return val
-
-    def setter_subname(self, val):
-        # On PUT, RRsetSerializer sends None, denoting the unchanged value
-        if val is None:
-            return self.subname
-
-        if val != self.subname:
-            self._dirties.add('subname')
-
-        return val
-
-    def setter_type(self, val):
-        if val != self.type:
-            self._dirties.add('type')
-
-        return val
+    @property
+    def name(self):
+        return '.'.join(filter(None, [self.subname, self.domain.name])) + '.'
 
-    def setter_ttl(self, val):
-        if val != self.ttl:
-            self._dirties.add('ttl')
+    def save(self, *args, **kwargs):
+        self.updated = timezone.now()
```

Comment:
Now the timestamp is updated even if the request was no-op, in terms of DNS content

Reply:
Yes, to fix that the change tracker needs to now old and new values of the RRset object (but I just removed the tracking of changes entirely). I'll re-introduce that code and then update. Also, I'll double check that there are tests covering the non-update of pdns and `self.updated` in case the value doesn't change.